RelhaxModpack/RelhaxModpack.sln

```diff
@@ -1,10 +1,12 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29806.167
+# Visual Studio 15
```

Comment:
Im guessing this is cause I upgraded to VS 2019. I found that in 2017 the unit tests would execute in the order of which they were created. Very annoying.

In 2019 they execute in alphabetical order. So i was able to make them like `Setxx_feature`